var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -45,7 +45,7 @@ class Lbann(CMakePackage):
     depends_on('elemental +openmp_blas +scalapack +shared +int64 +debug', when='+debug')
     depends_on('cuda', when='+gpu')
     depends_on('mpi')
-    depends_on('opencv@2.4.13', when='+opencv')
+    depends_on('opencv@3.2.0', when='+opencv')
```

Comment:
Just to confirm, is this change intended to be in this PR? It wasn't mentioned in the description so I wanted to make sure it didn't slip in accidentally.